ELISA Policies and Guidelines

```diff
@@ -0,0 +1,69 @@
+# ELISA Policies and Guidelines
+
+This document outlines the policies and guidelines for developing and publishing ELISA deliverables.
+
+**ELISA deliverables:**
+- ELISA develops an example qualitative analysis for safety critical use-cases for the Linux kernel e.g: Automotive and Medical.
+- ELISA provides resources for System integrators to apply and use to analyze qualitatively and quantitatively on their systems. For example, ELISA will analyze published CWEs to identify hazards for the two use-cases.
+- System Integrators use ELISA deliverables to analyze their systems
+
+**What are the publications:**
+- Whitepapers and blogs published on elisatech etc.
+- Link to the github repository
+
+Presentations and work in progress are not considered a publication until reviewed and added to one of the publication sites. All work in progress on github should contain licenses.
+
+### Publication Licensing Policy
+All ELISA documentation deliverables shall be released under CC-BY-4.0. All published documents shall carry the following licensing information.
```

Comment:
I have a concern here. If we say above that blogs and whitepapers count as documentation deliverable it will look strange to have the CC-BY-4.0 listed there. I believe for markdown however, this makes sense. There may be an exception if you have a documentation focused repository which include a LICENSE file.